linalg/operator.hpp

```diff
@@ -969,6 +969,10 @@ class TripleProductOperator : public Operator
 
    MemoryClass GetMemoryClass() const override { return mem_class; }
 
+   const Operator *GetA() const { return A; }
+   const Operator *GetB() const { return B; }
+   const Operator *GetC() const { return C; }
```

Comment:
Wonder if these should return `const Operator &` instead of `const Operator *`.  The main difference is that the pointer return value is allowed to return `nullptr` where references aren't allowed to be `nullptr`. The `const` makes it pretty clear that ownership of the pointer is *not* being transferred.

Reply:
Done in c3f7e1d7.